armory/metrics/poisoning.py

```diff
@@ -58,27 +62,41 @@ def __init__(
         data_modality="image",
         model_framework="pytorch",
         activation_layer=None,
-        resize_image=True,
-        size=(224, 224),
-        resample=Image.BILINEAR,
-        device=DEVICE,
+        preprocess_kwargs={},
```

Comment:
I suggest having `preprocess_kwargs` default to `None` rather than a dict. See, e.g., [here](https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/) for a description of potential issues with default mutable args

You could then do:
```
if preprocess_kwargs is None:
    preprocess_kwargs = {}
```
You should then be able to remove all the various
```
        "preprocess_kwargs": {},
```

that are in `EXPLANATORY_MODEL_CONFIGS`, which would be cleaner

Reply:
Agree, done